modules/microshift-stopping-lvms-csi-snapshot.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/microshift-storage-plugin-overview.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-stopping-lvms-csi-snapshot_{context}"]
+== Stopping LVMS and CSI volume snapshot deployment
+
+Workloads with attached volumes must be manually stopped, and those volumes must then be manually deleted by the user. Use the following procedure to prevent the deployment of LVMS CSI driver and CSI volume snapshot controller.
```

Comment:
MicroShift treats the deployment of the CSI driver and the CSI snapshot implementations independently. As said in comments below, "deployment" is probably too overloaded to be useful here.

I would suggest something like:
```suggestion
Before disabling the CSI driver or CSI snapshot implementation pods, any user workloads with attached volumes must be manually stopped. Those volumes must then be manually deleted by the user. MicroShift will not attempt to delete CSI implementation pods. Instead, MicroShift must be configured not to run one or both of the CSI pods during the start-up process. Use the following procedures to disable installation of one or both CSI components, then restart MicroShift.
```